packages/components/src/console/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useRef, ReactElement } from 'react';
+import { ConsoleMessage } from './ConsoleMessage';
+import { Message } from './ConsoleMessage';
+
+export const ConsoleMessages = ({ messages }: { messages: Message[] }): ReactElement => {
+  const AlwaysScrollToBottom = () => {
+    const elementRef = useRef<null | HTMLDivElement>(null);
+    useEffect(() => elementRef?.current?.scrollIntoView());
+    return <div ref={elementRef} />;
+  };
+
+  const Messages = () => {
+    return (
+      <>
+        {messages.map((message, index) => {
+          return <ConsoleMessage key={index} mIndex={index} message={message} />;
```

Comment:
Using the array's index as key should almost always be avoided. We might e.g. change the message context later on, to display only the last 20 messages and to drop older one's. In this case, the index wouldn't be a unique key. Instead, `message.id` should be used.